src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -317,7 +349,7 @@ public object GetOrRegisterObjectForComInstance(IntPtr externalComObject, Create
         /// <remarks>
         /// If <paramref name="impl" /> is <c>null</c>, the global instance (if registered) will be used.
         /// </remarks>
-        private static bool TryGetOrCreateObjectForComInstanceInternal(
+        private static unsafe bool TryGetOrCreateObjectForComInstanceInternal(
             ComWrappers impl,
```

Comment:
Can this be an instance method instead? I do not see the point of passing in impl explicitly. (CreateCCW has the same issue.)